plugins/org.jkiss.dbeaver.model.lsm/grammar/SQLStandardParser.g4

```diff
@@ -331,9 +331,9 @@ selectStatementSingleRow: SELECT (setQuantifier)? selectList INTO selectTargetLi
 selectTargetList: parameterSpecification (Comma parameterSpecification)*;
 sqlDataChangeStatement: (deleteStatement|insertStatement|updateStatement);
 deleteStatement: DELETE FROM tableName whereClause?;
-insertStatement: INSERT INTO tableName insertColumnsAndSource;
-insertColumnsAndSource: ((LeftParen insertColumnList RightParen)? queryExpression|DEFAULT VALUES);
-insertColumnList: columnNameList;
+insertStatement: INSERT INTO nonjoinedTableReference? insertColumnsAndSource;
+insertColumnsAndSource: ((LeftParen (insertColumnList?|Asterisk) RightParen?)? queryExpression|DEFAULT VALUES)?;
+insertColumnList: columnReference (Comma columnReference)*;
```

Comment:
There cannot be qualified names in the insertColumnList, there are only identifiers always, so there is no point in trying to parse columnReference looking for a dot character - this will slow down parsing.